### PR TITLE
Wrap conversations in links

### LINKED ISSUE #2684: Navigation conversations is not a proper link
On the left navigation, the conversations are not links, that means that:
1. you cannot use the keyboard to properly navigate to it
2. you cannot long-press/right-click to open in a new tab

Capture d’écran_2020-01-07_15-04-26

### PR DESCRIPTION
**Pull request by ma12-co**
Fix #2684

how to test: 
* navigate conversations with keyboard;
* open conversations in new tabs;

Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue [hunk 1]
```diff
@@ -21,40 +21,42 @@
...
+			class="acli"
+			@click="onClick">
+			<!-- default slot for avatar or icon -->
+			<slot name="icon" />
+			<div class="acli__content">
+				<div class="acli__content__line-one">
+					<span class="acli__content__line-one__title">
+						{{ title }}
+					</span>
+					<Actions
```
Comment (skjnldsv) [inline src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue]: Actions cannot be inside the link I think, I can already see issues with it :thinking: 
The link should probably only cover the icon and titles?

Comment (ma12-co) [reply src/components/LeftSidebar/ConversationsList/AppContentListItem/AppContentListItem.vue]: The inner <a> (actions) seems to have precedence over the grandparent <a>, so far I tested this with chrome and firefox, with clicking, tapping and keyboard navigation and it all works good. I'm looking for documentation on this but I can't find much.

**Comment by ma12-co** [comment]
ready for review!

**Comment by nickvergessen** [comment]
Regression, all conversation icons are now color-inverted again on dark-mode, because the server rule matches:
```
#app-navigation > ul > li > a:first-child img, #app-navigation > ul > li > ul > li > a:first-child img, #contactsmenu-menu a img, #expanddiv a img, .activity-section .activity-icon.monochrome img
```

proposal would be to wrap it with a
```scss
:not(.no-icon-invert) {
}
```

**Pull request merged**